vmdb/app/controllers/ops_controller/db.rb

```diff
@@ -230,11 +230,8 @@ def db_get_info(nodetype)
   end
 
   def get_indexes(tb)
-    indexes = Array.new
-    tb.vmdb_indexes.sort{|a,b| a.name <=> b.name}.each do |idx|
-      indexes.push(idx) if idx.vmdb_table.type == "VmdbTableEvm"
-    end
-    return indexes
+    return [] unless tb.kind_of?(VmdbTableEvm)
+    tb.vmdb_indexes.sort_by(&:name)
```

Comment:
This one is interesting.  So the callers to get_indexes are [here](https://github.com/Fryguy/manageiq/blob/replace_sort_with_sort_by_4/vmdb/app/controllers/ops_controller/db.rb#L206-L207) and [here](https://github.com/Fryguy/manageiq/blob/replace_sort_with_sort_by_4/vmdb/app/controllers/ops_controller/db.rb#L224-L225).

In both cases, we get a VmdbTable instance and pass it into the method.  The code here previously asked for the indexes of that table, and then checked the parent table of each of the indexes again via SQL.  But the code *is passing in the table* already, so there's not need to go back to the database to recheck it.  So, it can just check right up front if it's from the right table type or not.  After that, it's a simple `.sort_by`

This might be a significant performance boost because the old code did A LOT of SQL.